scripts/build/termux_setup_self_debug.sh

```diff
@@ -0,0 +1,15 @@
+termux_print_stacktrace() {
+	local func src lineno
+	echo "💥 Error occurred at line $1 in script ${BASH_SOURCE[1]}"
+	echo "🔍 Stack trace:"
+	for i in "${!FUNCNAME[@]}"; do
+		func="${FUNCNAME[$i]}"
+		src="${BASH_SOURCE[$i]}"
+		lineno="${BASH_LINENO[$((i - 1))]}"
+		# skip print_stacktrace itself
+		(( $i )) && echo "  at $func() in $src:$lineno"
```

Comment:
[`SC2004, $/${} is unnecessary on arithmetic variables.`](https://www.shellcheck.net/wiki/SC2004)
```suggestion
		(( i )) && echo "  at $func() in $src:$lineno"
```
Also why is there two spaces in front of this line?

Reply:
To have a nice indentation.